pkg/framework/host.go

```diff
@@ -61,11 +62,20 @@ Installation:
 )
 
 type Host struct {
+	backoff   *backoff.ExponentialBackOff
 	g8sClient *giantclientset.Clientset
 	k8sClient kubernetes.Interface
 }
 
-func NewHost() (*Host, error) {
+type Config struct {
+	Backoff *backoff.ExponentialBackOff
+}
+
+func NewHost(c *Config) (*Host, error) {
+	if c.Backoff == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.Backoff must not be empty", c)
+	}
```

Comment:
You could use a default one here instead of failing.

Reply:
Nice, done.